breaking-front/src/pages/ProfileEdit/ProfileEdit.js

```diff
@@ -1,35 +1,100 @@
-import React from 'react';
-import { useMutation } from 'react-query';
+import React, { useContext, useState } from 'react';
+import { useMutation, useQueryClient } from 'react-query';
 import { useNavigate } from 'react-router-dom';
 import ProfileSettingForm from 'components/ProfileSettingForm/ProfileSettingForm';
 import useProfileDetailData from 'hooks/queries/useProfileDetailData';
 import { putProfileEdit } from 'api/profileEdit';
 import { PAGE_PATH } from 'constants/path';
+import * as Style from 'pages/ProfileEdit/ProfileEdit.styles';
+import Line from 'components/Line/Line';
+import Modal from 'components/Modal/Modal';
+import Button from 'components/Button/Button';
+import { deleteProfileWithdrawal } from 'api/profile';
+import { UserInformationContext } from 'providers/UserInformationProvider';
 
 const ProfileEdit = () => {
   const navigate = useNavigate();
-  const { data, isLoading } = useProfileDetailData();
-
-  const { mutate: ProfileEditMutate } = useMutation(putProfileEdit, {
-    onSuccess: (res) => {
-      console.log(res);
-      alert('변경사항을 저장하였습니다.');
-      navigate(PAGE_PATH.HOME);
-    },
-    onError: () => {
-      //에러 페이지 이동
-    },
-  });
+  const queryClient = useQueryClient();
+  const { userId } = useContext(UserInformationContext);
+
+  const { data: profileData, isLoading: isProfileDataLoding } =
+    useProfileDetailData();
+  const [isOpenWithdrawal, setIsOpenWithdrawal] = useState(false);
+  const [isChecked, setIsChecked] = useState(false);
+
+  const { mutate: ProfileEditMutate, isLoading: isProfileEditLoading } =
+    useMutation(putProfileEdit, {
+      onSuccess: (res) => {
+        alert('변경사항을 저장하였습니다.');
+        queryClient.invalidateQueries('initalizeValidUser');
```

Comment:
```suggestion
        queryClient.invalidateQueries('initializedValidUser');
```
userInformationProvicer.js
financial.js
에도 이렇게 되어있네요